index.js

```diff
@@ -35,5 +35,15 @@ export default async function taskkill(input, options = {}) {
 		arguments_.push(typeof element === 'number' ? '/pid' : '/im', element);
 	}
 
-	await execa('taskkill', arguments_);
+	return arguments_;
 }
+
+export function taskkillSync(input, options = {}) {
+	execaSync('taskkill', parseArgs(input, options));
+}
+
+export async function taskkill(input, options = {}) {
+	execa('taskkill', parseArgs(input, options));
```

Comment:
You have made a floating promise, which means the action would not be awaited and error could not be handled. Please add `await`.